### DISCUSSION TITLE
Looking at the supported list of devices, I don't see the ESP8266 Wemos D1 and similar listed?

### DISCUSSION THREAD

**Discussion by Jibun-no-Kage** [arendst/Tasmota#19827] [Q&A] [answered]
Looking at the supported list of devices, I don't see the ESP8266 Wemos D1 and similar listed?  But the ESP-01/01s, ESP-12 variants are.  I discovered that the older ESP8266 dev. modules I have, which are a bit older, some to fail to connect on TAS firmware 13.1/.2, but anything older than 13.1 seems to work still.  For example TAS 9.3.1 is rock solid on the older modules I have.  Wondering if this a known issue?

00:00:00.049 CFG: Loaded from flash at F4, Count 8
00:00:00.055 QPC: Count 1
00:00:00.060 Project tasmota Tasmota Version 9.3.1(lite)-2_7_4_9(2021-03-09T16:12:49)
00:00:00.548 WIF: Connecting to AP1 DD Channel 11 BSSId XX:XX:XX:XX:XX:XX in mode 11n as tasmota_8FF589-5513...
00:00:03.756 WIF: Connected
00:00:04.008 HTP: Web server active on tasmota_8FF589-5513 with IP address 192.168.1.140
19:11:23.204 RSL: tele/tasmota_8FF589/INFO1 = {"Module":"Sonoff Basic","Version":"9.3.1(lite)","FallbackTopic":"cmnd/DVES_8FF589_fb/","GroupTopic":"cmnd/tasmotas/"}
19:11:23.209 RSL: tele/tasmota_8FF589/INFO2 = {"WebServerMode":"Admin","Hostname":"tasmota_8FF589-5513","IPAddress":"192.168.1.140"}
19:11:23.220 RSL: tele/tasmota_8FF589/INFO3 = {"RestartReason":"External System"}
19:11:23.228 RSL: stat/tasmota_8FF589/RESULT = {"POWER":"OFF"}
19:11:23.233 RSL: stat/tasmota_8FF589/POWER = OFF
19:11:24.455 QPC: Reset
19:11:28.378 RSL: tele/tasmota_8FF589/STATE = {"Time":"2023-10-23T19:11:28","Uptime":"0T00:00:12","UptimeSec":12,"Vcc":3.039,"Heap":32,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"POWER":"OFF","Wifi":{"AP":1,"SSId":"DD","BSSId":"XX:XX:XX:XX:XX:XX","Channel":11,"RSSI":100,"Signal":-49,"LinkCount":1,"Downtime":"0T00:00:05"}}

Upgrade...
19:17:07.548 RSL: stat/tasmota_8FF589/RESULT = {"Upgrade":"Version 9.3.1 from http://ota.tasmota.com/tasmota/release/tasmota.bin.gz"}
19:17:10.174 HTP: Web server stopped
19:17:22.216 HTP: Web server active on tasmota_8FF589-5513 with IP address 192.168.1.140
19:17:23.469 RSL: stat/tasmota_8FF589/UPGRADE = {"Upgrade":"Successful. Restarting"}

Whereas...
 ets Jan  8 2013,rst cause:2, boot mode:(3,0)

load 0x4010f000, len 3584, room 16
tail 0
chksum 0xb0
csum 0xb0
v3969889e
@cp:0
     ld

00:00:00.001 HDW: ESP8266EX
00:00:00.046 CFG: Loaded from flash at FB, Count 10
00:00:00.156 QPC: Reset
00:00:00.176 Project tasmota - Tasmota Version 13.2.0(tasmota)-2_7_4_9(2023-10-19T09:02:07)
00:00:00.808 WIF: Connecting to AP1 DD Channel 11 BSSId XX:XX:XX:XX:XX:XX in mode 11n as tasmota-8FF589-5513...
00:00:10.532 WIF: Connect failed as AP cannot be reached
00:00:10.870 WIF: Connecting to AP1 DD in mode 11n as tasmota-8FF589-5513...
00:00:25.023 WIF: Connect failed with AP timeout

The above is an endless loop, for over an hour in one test.  So where 9.3.1, 12.0.2, 12.0.5, etc. even just short of 13.1, seems to work.  But 13.0.1 did not work well, it sometimes failed to find AP.  Maybe a code change with 13.0... made the older ESP8266 modules obsolete or abandoned?  They are pretty old, but still functional.

**Comment by s-hadinger** [comment]
No this is not a known problem. Wemos D1 mini are well supported and I have numerous such devices. 

We need to investigate

**Comment by Jason2866** [comment]
Just updated two of my WemosD1 minis. The are running Tasmota since 4 years and updated now and than
(just temp humi sensor connected). So it does not really matter if latest version is installed...
But the WemosD1 minis made a lot of trouble until the day where I replaced the underrated 150mA 3.3Volt LDO against 500mA types.

**Comment by Jibun-no-Kage** [comment]
Now that is interesting!  In my case, anything flashed below 13.x seems solid, but anything 13.x, I see the AP connection issues.  Wonder is this an echo of the old SDK issue(s) theory I recall discussed in the past.  I noted in another thread, that I can live with 12.x for my needs for now.  But I did wonder if what I was seeing was a known issue, or just my experience.  I use a good PS, 5v 4a stepped down to 3.3v for flashing, then as is for testing.  My DHCP server seems fine, I restarted it just to rule that out.  My other devices are working great, I have Pis using WiFi, ESP-32s, ESP-01/S modules all running 13.x fine, such, so don't think this is a WiFi router/AP issue.  But if I let one of these older modules, flashed with 13.2... it either never connects or only does after 10s, sometimes 100s of attempts.  Same behavior trying to connect to 192.168.4.1 initially, all kinds of trouble to setup WiFi initially.

**Reply by stefanbode** [reply]
It is widely known that with the change from 12 to 13 the power consumption increases. This causes trouble on some cheap devices or clones. The "lite" version sometimes helps to get it working. Also super oversized capacitors helps in some cases. 

I keep some devices on 12. the rest I exchanged the LDO. This works fine. My wemos D2 so far had no issues. All powered by 5V. Hope this give some ideas how to fix. A code change is very unlikely, beside the already optimized "lite" version.

**Comment by Jason2866** [comment, marked as answer]
On the working device you used the lite version `Tasmota Version 9.3.1(lite)` the lite is build with only a few features and needs less power. Now it is updated to the full version which has more features and needs more power than the lite version. You can try the actual lite version
A good 5V power supply does not help. The problem "sits" on the PCB. The weak 3.3V LDO which regulates the 5V to 3.3V. Replace this part on the PCB against a 500mA type and the device **WILL** work
All esp have massive wifi issues when power supply is bad.

**Comment by Jibun-no-Kage** [comment]
As to lite versus full, that was just an artifact of the testing reporting.  The testing I did today was both with lite and full, and  there was no difference.  I don't disagree, with anything suggested, only to point out the fact that 9.3.1 lite and full, are solid on the same hardware, and that 12.x works well.  It only when I try 13.x I see the issue of the AP connection inconsistency results.  

Today I did more testing, and the results are consistent.  IMHO, using a good power supply does help, in the fact, that it rules an obvious and typical issue out, of course the PCB age/poor component quality can and would be applicable, and I would agree completely, if not for the results I noted above., call it a surprise.  It would not surprise me, if the reason 9.3.1 seems to perform so well, is that it just is not as demanding code?  So it fails to push the older modules with their older/poorer components?  

If nothing else, this discussion should provide insight to future issues of a similar nature.  I have never expected a code change, only to raise awareness, so that if there was something a miss it would be addressed.  I believe I noted above, I expect to leave these modules at 12.x, that is more than sufficient for my needs.
